src/applications/disability-benefits/4142/helpers.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+export const aboutPrivateMedicalRecs = () => {
+  return (
+    <div>
+      <h4>About private medical records</h4>
+      <p>
+        You said you were treated for [condition] by a private doctor. If you
```

Comment:
`[condition]` is meant to be filled in later, right?

Reply:
Yes. After integration with 526, [condition] will be filled in.